client/my-sites/checkout/checkout/composite-checkout-payment-methods.tsx

```diff
@@ -111,10 +112,10 @@ export function createPaymentMethods( {
 							siteId: select( 'wpcom' )?.getSiteId?.(),
 							domainDetails: getDomainDetails( select ),
 							couponId: null, // TODO: get couponId
-							country: select( 'wpcom' )?.getContactInfo?.()?.country?.value,
+							countryCode: select( 'wpcom' )?.getContactInfo?.()?.countryCode?.value,
```

Comment:
This is submitting `countryCode` to `makePayPalExpressRequest()`, but that function is looking for `country`, so it looks like this is still an issue.